src/gql/ProgramDonorSummary/resolvers/types.ts

```diff
@@ -190,6 +190,7 @@ export type DonorSummaryEntrySort = {
   order: SortOrder;
 };
 export type BaseQueryArguments = {
+  rdpcCode: string;
```

Comment:
This service is running in a single rdpc so can only do indexing in that single data center, so the rdpc for this server should be an env config property and not needed in the request inputs.